docs/source/_src_user_guide/state_propagation/propagation_setup.rst

```diff
@@ -44,14 +47,43 @@ in Tudat we also have:
    dynamics, for instance, there are various options besides a simple Cartesian state representation. However, even
    when using a non-Cartesian state vector, the Cartesian representation still plays a role in calculating, *e.g.*,
    acceleration models, as well as in defining the initial state. For more information on the role of different state
-   representations, please visit the page :ref:`propagator_types`.
+   representations, please visit the page :ref:`conventional_propagated_states`.
 
 
-Inputs
-======
+The above list defines different types of dynamics that are propagated over a single continuous arc.
+Propagation using a :ref:`multi-arc setup <multi_arc_dynamics>` is also supported in Tudat.
 
-As the figure shows, there are some input arguments common to all types of dynamics, while some others are
+.. _propagation_inputs:
+
+Inputs and setup
+================
+
+As the figure shows, there are some input arguments common to all types of single-arc dynamics, while some others are
 specific to the type of propagator. More information about inputs are explained in the dynamic-specific pages linked
 above.
 
 .. figure:: propagation_setup/_static/tudatpy_propagation_settings.png
+
+The propagation of a given type of dynamics is defined by calling the
+`associated factory function <https://py.api.tudat.space/en/latest/propagator.html#functions>`_.
+For the different types of single are dynamics, these factory functions return an object (derived from the
+:class:`~tudatpy.numerical_simulation.propagation_setup.propagator.SingleArcPropagatorSettings` class)
+defining the settings of the propagation. The specifics of how to change the settings for the different
+types of dynamics are discussed on the pages linked to above.
+
+There are various options in the propagator settings that can be provided by a user to modify the behaviour before, during and after
```

Comment:
It says 'can be provided', but is there a default numerical integrator? Or do you have to provide a numerical integrator, in which case the formulation could be improved. I thought you always had to provide IntegratorSettings.

Reply:
Good point. Change to "must (or can)". For some settings, there are defaults